Packaging.props

```diff
@@ -12,6 +12,8 @@
     <PackagePlatform Condition="'$(PackagePlatform)' == 'amd64'">x64</PackagePlatform>
     <NativePackagePath>$(MSBuildThisFileDirectory)src/Native/pkg</NativePackagePath>
   </PropertyGroup>
+  
+  <Import Project="pkg/Microsoft.Private.PackageBaseline/Microsoft.Private.PackageBaseline.props" />
```

Comment:
What is your plan for consuming this in other repo's like WCF? I would like to make sure they don't have to do anything too crazy.